analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -2,132 +2,110 @@
 using System.Collections.Generic;
 using System.Linq;
 
-public class EmptyNullableValueAccess
+class EmptyNullableValueAccess
 {
-    private IEnumerable<TestClass> numbers = new[]
+    private IEnumerable<TestClass> Numbers = new[]
     {
-        new TestClass { Number = 42 },
+        new TestClass() { Number = 42 },
         new TestClass(),
-        new TestClass { Number = 1 },
-        new TestClass { Number = null }
+        new TestClass() { Number = 1 },
+        new TestClass() { Number = null }
     };
 
-    public class TestClass
+    void NullAssignment()
     {
-        public int? Number { get; set; }
-    }
-
-    void Nameof(object o)
-    {
-        if (o == null)
+        int? i = null;
+        if (i.HasValue)
         {
+            Console.WriteLine(i.Value);
         }
 
-        if (o == nameof(Object))
-        {
-            o.ToString(); // Compliant
-        }
+        Console.WriteLine(i.Value); // Noncompliant {{'i' is null on at least one execution path.}}
+        //                ^
     }
 
-    public void TestNull()
-    {
-        int? i1 = null;
-        if (i1.HasValue)
-        {
-            Console.WriteLine(i1.Value);
-        }
-
-        Console.WriteLine(i1.Value); // FIXME Non-compliant {{'i1' is null on at least one execution path.}}
-        // FIXME          ~~~~~~~~
-    }
+    IEnumerable<TestClass> EnumerableExpressionWithCompilableCode() =>
+        Numbers.OrderBy(x => x.Number.HasValue).ThenBy(i => i.Number);
 
-    public IEnumerable<TestClass> TestEnumerableExpressionWithCompilableCode() => numbers.OrderBy(i => i.Number.HasValue).ThenBy(i => i.Number);
-    public IEnumerable<int> TestEnumerableExpressionWithNonCompilableCode() => numbers.OrderBy(i => i.Number.HasValue).ThenBy(i => i.Number).Select(x => x.Number ?? 0);
+    IEnumerable<int> EnumerableExpressionWithNonCompilableCode() =>
+        Numbers.OrderBy(x => x.Number.HasValue).ThenBy(i => i.Number).Select(x => x.Number ?? 0);
```

Comment:
Same as above. But it also confuses me: This compiles for me, so why the name?

Reply:
I don't know, it was there in the previous testing code (for the Sonar version of the symbolic execution), and I hadn't removed it.